library-best-practices/JLBP-11.md

```diff
@@ -0,0 +1,7 @@
+[JLBP-11] Frequent release cadence for non-breaking changes
+-----------------------------------------------------------
+
+- Keep compatible dependencies up-to-date.
+- Recommendation: Perform a release at least once every 6-weeks.
```

Comment:
Perform a release --> Release

Every six weeks is very quick for projects in maintenance mode. I might suggest "at least twice a year and preferably every two months".

How feasible this is depends a lot on the automation of the build, test, release process. I wonder if we should say something about that?

Reply:
I think projects in maintenance mode either need to be pulled out of the tree altogether, or actually have dependencies kept up to date. Otherwise, it's a huge source of pain for the ecosystem. (See: google-*-client-java libraries).